pkg/flow/internal/controller/loader_test.go

```diff
@@ -124,15 +124,8 @@ func TestLoader(t *testing.T) {
 		require.Error(t, diags.ErrorOrNil())
 
 		requireGraph(t, l.Graph(), graphDefinition{
```

Comment:
see [here](https://github.com/grafana/agent/pull/3297/files#r1139349663) for context